src/views/v2/emails/template-parts/body/ticket/holder-name.php

```diff
@@ -25,11 +25,11 @@
  * @see tribe_get_event() For the format of the event object.
  */
 
-if ( empty( $ticket['holder_name'] ) ) {
+if ( empty( $ticket->holder_name ) ) {
```

Comment:
This and all the template changes will make the tests pass, but it'll break the emails, as `$ticket` is an array.